.github/workflows/publish-pypi.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       url: https://pypi.org/p/stravalib
     permissions:
       id-token: write  # Mandatory for pypi publishing
-     
```

Comment:
This is a product of running precommit across all files and is unrelated to this pr but is needed to ensure this pr passes CI 🤷🏻‍♀️